apps/portal/examples/postgraphile-template/content/source/README.md

```
@@ -0,0 +1,146 @@
+touch db/migrations/000002_add_profile_bio.down.sql
+
+# 2. Write your schema alteration statement
+echo "ALTER TABLE public.users ADD COLUMN bio TEXT;" >> db/migrations/000002_add_profile_bio.up.sql
+
+# 3. Push to your main branch
+git add db/migrations/
+git commit -m "feat: add user bio column"
+git push origin main
+```
+
+The underlying `db-migrate` execution network automatically applies the updates in the background, instantly updating your `/graphiql` schema tree with zero user downtime!
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)